std/experimental/allocator/package.d

```diff
@@ -748,31 +748,40 @@ unittest
 }
 
 /// Ditto
-T[] makeArray(T, Allocator, R)(auto ref Allocator alloc, R range)
+auto makeArray(T = void, Allocator, R)(auto ref Allocator alloc, R range)
```

Comment:
I would prefer that, but
1) this might break compatibility (which might be okay because it's experimental)
2) removes to specify the feature to use a different type